app/src/test/java/org/simple/clinic/newentry/form/DateOfBirthFieldTest.kt

```diff
@@ -19,4 +25,10 @@ class DateOfBirthFieldTest {
     assertThat(dateOfBirthField.validate("7/6"))
         .containsExactly(InvalidDateFormat)
   }
+
+  @Test
+  fun `it returns date is in future when date of birth is in the future`() {
+    assertThat(dateOfBirthField.validate("12/12/2020"))
```

Comment:
When it comes to tests like these, I prefer to keep the date exactly one minimal unit after the limit to verify that it's working as expected.

i.e, when verifying that a given `LocalDate` (say `a`) is not more than another `LocalDate` (say `b`), I change `a` to be exactly one day after `b`. If it was an `Instant`, it would be 1 millisecond instead.

And then have other tests to verify that the check succeeds on the same/earlier timestamp. What do you think?

Reply:
That makes sense. Fixed it!